strax/context.py

```diff
@@ -1404,14 +1404,19 @@ def _apply_function(self, data, targets):
             data = function(data, targets)
         return data
 
-    def copy_to_frontend(self, run_id, target,
-                         target_frontend_id=None, rechunk=False):
+    def copy_to_frontend(self,
+                         run_id: str,
+                         target: str,
+                         target_frontend_id: int = None,
+                         target_compressor: str = None,
```

Comment:
Is it worth considering passing non-null but type-correct "empty" default args here? So an empty string for the target_compressor and something equivalent for target_frontend_id?

Reply:
sure, let me change the typecasting to `typing.Optional[str]`

Reply:
Alright, fixed in f8e0809 

See typing.Optional for the documentation: https://docs.python.org/3/library/typing.html#typing.Optional